Sources/PushNotifications.swift

```diff
@@ -190,6 +182,28 @@ import Foundation
         return persistenceService.getSubscriptions()
     }
 
+    /**
+     Handle Remote Notification.
+
+     - Parameter userInfo: Remote Notification payload.
+     */
+    @objc public func handleNotification(userInfo: [AnyHashable: Any]) {
```

Comment:
I think this could be returning a boolean so the customer code would do:
```

      func application(_ application: NSApplication, didReceiveRemoteNotification userInfo: [String : Any]) {
         if (self.pushNotifications.handleNotification(userInfo: userInfo)) {
            return;
         }
      }
      
```

Reply:
Separate PR 👍